kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -104,13 +104,23 @@ class FlinkProcessBuilder(
         val userJars = conf.get(ENGINE_FLINK_APPLICATION_JARS)
         userJars.foreach(jars => flinkExtraJars ++= jars.split(","))
 
+        val hiveOptions = env.get("HIVE_CONF_DIR")
+        if (hiveOptions.isDefined) {
+          val hiveConf = s"${hiveOptions.get}${File.separator}hive-site.xml"
+          flinkExtraJars += hiveConf
+        }
```

Comment:
nit: a style suggestion

```suggestion
        val hiveConfDirOpt = env.get("HIVE_CONF_DIR")
        hiveConfDirOpt.foreach { hiveConfDir =>
          val hiveConf = s"${hiveConfDir}${File.separator}hive-site.xml"
          flinkExtraJars += hiveConf
        }
```